CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+We use the [Haskell Foundation's Guidelines For Respectful
+Communication](https://haskell.foundation/guidelines-for-respectful-communication/)
```

Comment:
This document contains early on:
> Diversity and inclusion. We recognize that the Haskell community, echoing the technology industry more generally, skews white and male.

Such claims should be backed by data.  Also, I do not see how this implicit _white-male_ shaming helps with the goal of _inclusion_.

Reply:
https://taylor.fausak.me/2021/11/16/haskell-survey-results/#s7q1